app/games/[gameName]/scoreboard/index.tsx

```diff
@@ -0,0 +1,178 @@
+import { useLocalSearchParams } from "expo-router";
+import { View, ScrollView } from "react-native";
+import { Text } from "react-native-paper";
+
+import { actions } from "@/common/actions";
+import { useQuery } from "@/common/api";
+import { useI18n } from "@/common/i18n";
+import { RedirectIfNoProfile, useIdentity } from "@/common/identity";
+import { capitalize } from "@/common/string";
+import { sty } from "@/common/styles";
+import { Header, LoadingScreen } from "@/components";
+
+const Scoreboard = () => {
+  const { t } = useI18n();
+  const styles = useStyles();
+  const identity = useIdentity();
+  const { gameName } = useLocalSearchParams();
+  const { data: leaderboardData } = useQuery(
+    gameName && typeof gameName === "string"
+      ? {
+          url: "/api/v1/games/{gameName}/leaderboard",
+          params: { path: { gameName }, query: { limit: 10 } },
+        }
+      : null,
+  );
+  const { data: yourScoreData } = useQuery(
+    gameName && typeof gameName === "string"
```

Comment:
```suggestion
  const { data: leaderboardData } = useQuery(
    typeof gameName === "string"
      ? {
          url: "/api/v1/games/{gameName}/leaderboard",
          params: { path: { gameName }, query: { limit: 10 } },
        }
      : null,
  );
  const { data: yourScoreData } = useQuery(
    typeof gameName === "string"
```
nit: wouldn't that be enough? (I'm not sure)

Reply:
youre right